pkg/ccl/streamingccl/streamingest/alter_replication_job.go

```diff
@@ -195,6 +194,7 @@ func alterReplicationJobHook(
 	return fn, nil, nil, false, nil
 }
 
+// alterTenantJobCutover returns the actual cutover timestamp used.
```

Comment:
nit: actual is a little ambiguous, could we expand this comment to spell out what timestamp will be returned?

Reply:
done.